migrations/215_files__add_storage_type.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE files 
+    ADD COLUMN storage_type text DEFAULT 'fileSystem',
```

Comment:
```suggestion
    ADD COLUMN storage_type enum_file_storage_type DEFAULT 'fileSystem',
```
Would this make sense as a formal enum? (defined in `database/enums/enum_file_storage_type.pg`)

Reply:
I think that is way better too!  I'll do that.